src/resources/swagger.yaml

```diff
@@ -178,10 +178,10 @@ paths:
             application/json:
               schema:
                 type: object
-  /submission/schema/list:
+  /dictionary/list:
```

Comment:
I meant to open a ticket for gateway and some one will do that later, not for you to do it now you can keep the old path and just deprecate it

Reply:
even if we are going to do it right away it's still better to do it in two steps to avoid the overhead of coordinating deployments

Reply:
Agreed. I'm chaning in swagger though just because it will be messy to keep both and they are the same just with differnet path.

Reply:
swagger shows you deprecated if you mark them with deprecated

Reply:
yes but you have to keep the whole description. I can put it back if you prefer but it'll be the same things just different paths...

Reply:
but yeah no need to show them in swagger no one is checking them anyway